plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -84,7 +84,9 @@
 public class LocalAppEngineServerLaunchConfigurationDelegate
     extends AbstractJavaLaunchConfigurationDelegate {
 
-  private final static Logger logger =
+  private static final boolean DEVAPPSERVER2 = false;
```

Comment:
Good catch. The program is actually dev_appserver2.py (no break between app and server) so I changed it to DEV_APPSERVER for consistency.